portfolio/src/main/webapp/calendar.js

```diff
@@ -36,7 +38,11 @@ function createNewCalendarEvent() {
   const endTime =
       getTimeObject(document.getElementById('new-event-end-time').value);
 
-  const eventName = document.getElementById('new-event-name').value;
+  // If user enters an empty event name, sets the name to a default string.
+  let eventName = document.getElementById('new-event-name').value;
+  if (!eventName) {
+    eventName = 'New Event';
```

Comment:
Consider trying to make 'New Event' some class constant. Not sure how to do it in a Javascript class like this, but maybe you can do a bit of research to see whether it's possible.

Reply:
Maybe you can have constants.js that exports some constants, for example.

Reply:
Updated. I added a constant variable at the beginning of this calendar.js file.